.gitignore

```diff
@@ -32,7 +32,7 @@ builds/
 builds.zip
 docs/jsdocs
 
-development/bundle.js
+development/bundle.js*
```

Comment:
What is this change for? I don't see any files that this change would apply to

Reply:
On review, it was a temp file that was probably left over when I interrupted some script. I amended the commit.